js/src/collapse.js

```diff
@@ -332,14 +332,14 @@ class Collapse extends BaseComponent {
   // Static
 
   static collapseInterface(element, config) {
-    let data = Data.get(element, DATA_KEY)
+    let data = Collapse.getInstance(element)
     const _config = {
       ...Default,
       ...Manipulator.getDataAttributes(element),
       ...(typeof config === 'object' && config ? config : {})
     }
 
-    if (!data && _config.toggle && typeof config === 'string' && /show|hide/.test(config)) {
+    if (!data && _config.toggle && typeof config === 'string' && ['show', 'hide'].includes(config)) {
```

Comment:
Please revert this an any similar (regex to includes or equal to includes) changes in your PRs. IIRC the check is inverted now. Better keep PRs clean, one change at a time :)

Reply:
I can revert it as it is not on the scope of my Pr, and you are right,, but I am not sure if it is right practice. we can discuss it in another PR